lib/web/sessions.go

```diff
@@ -817,7 +835,7 @@ func (s *sessionCache) watchWebSessionsOnce(ctx context.Context, reset func()) e
 
 			s.log.
 				WithField("event", event).
-				Debug("Received sessionCache watcher event")
+				Trace("sessionCache: Received watcher event")
```

Comment:
When does `Trace`-level messages get logged in the context of the proxy service?

[slog doesn't seem to support `trace` as a level](https://pkg.go.dev/log/slog#Level), so I wonder if it's worth switching log statements from debug to trace.

Reply:
Trace-level is enabled by [log config stanza][1] (teleport.log.severity=trace).

slog works on a int-based level system. Trace isn't a named level, but in our case any level < LevelDebug is considered trace (see [lib/utils/log.TraceLevel][1]).

All of this is to say: this log statement isn't super useful, as whenever we actually process a message we will log the outcome. I only used it to make sure we were receiving events.

[1]: https://goteleport.com/docs/reference/config/#instance-wide-settings
[2]: https://github.com/gravitational/teleport/blob/bdba571a2961c7d589d8e1cef5eb511a53ac6ca2/lib/utils/log/slog_handler.go#L41

Reply:
Thanks for the explanation!